Guerrero_Melissa/Assignments/coin_tosses.py

```diff
@@ -0,0 +1,21 @@
+heads_count = 0
+tails_count = 0
+for count in range (0, 5001):
+	import random
+	random_num = random.random()
+	x = random_num
+	x_rounded = round(x)
+	
+
+	if (0 <= x_rounded <= 0.49):
```

Comment:
x = random.random() creates a number between 0 and 1, and stores it into x
x_rounded = round(x) will round that to the nearest integer (e.g. 0 -or- 1)

so here, you can check directly, if x_rounded==0:, and if x_rounded==1: